src/base/Random.cc

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (C) 1996-2022 The Squid Software Foundation and contributors
+ *
+ * Squid software is distributed under GPLv2+ license and includes
+ * contributions from numerous individuals and organizations.
+ * Please see the COPYING and CONTRIBUTORS files for details.
+ */
+
+#include "base/Random.h"
+
+std::mt19937::result_type
+Seed32()
```

Comment:
I am a little dubious about the name of this and the 64--bit variant. IMO it would be better called "RandomSeed*" to clarify that it is not a fixed value seed or such.

Reply:
Done (1bc3f7a).